scripts/cypress/cypress.config.ts

```diff
@@ -10,6 +10,19 @@ const applyV8WebpackConfig: (config: Configuration) => Configuration = require('
 
 const isLocalRun = !process.env.DEPLOYURL;
 
+function isV8() {
```

Comment:
context aware check instead of hard-coding path. 

_refactor later:_ ideally this shouln't be here but somewhere else

Reply:
probably should be some kind of utility like `isV8ExecutionContext`